### PR TITLE
Lithuanian translation

### PR DESCRIPTION
**Pull request by valius-kria**
Hello!

I am presenting my work of translation.  I'm sure, that it is possible in many places to write smoothier, but nobody of  my colleagues had time to reread everything.  So, if somebody will want to correct something, this can be done in the main site, I guess.

Valentinas

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -97,4 +103,5 @@ defaults:
       path: "ca"
     values:
       lang: "ca"
-      label: "Català"
\ No newline at end of file
+      label: "Català"
+theme: jekyll-theme-minimal
```
Comment (josephwright) [inline _config.yml:107]: Stray line?

**Review by josephwright — APPROVED** [review]

**Review by davidcarlisle — APPROVED** [review]
apart from line commented by Joseph look good, thanks

File: _config.yml (hunk 1 above)
Comment (valius-kria) [reply _config.yml]: Sorry, I do not remenber, why added the theme.  May be I didn't like the site look in my browser.

**Comment by davidcarlisle** [comment]
this is live at https://www.learnlatex.org/lt/  thanks!

**Pull request merged**